lib/linguist/languages.yml

```diff
@@ -1904,7 +1904,10 @@ Hack:
   codemirror_mode: php
   codemirror_mime_type: application/x-httpd-php
   extensions:
+  - ".hack"
+  - ".hck"
```

Comment:
Usage for that extension is [still too low](https://github.com/search?q=extension%3Ahck+NOT+sdhdfjdhfjd&type=Code). Could you remove it?